apps/etterlatte-saksbehandling-ui/client/src/components/person/saksoversikt.tsx

```diff
@@ -77,6 +78,15 @@ export const Saksoversikt = ({ fnr }: { fnr: string | undefined }) => {
     )
   }
 
+  const iverksatteBehandlinger = behandlingliste.filter(
+    (behandling) => behandling.status === IBehandlingStatus.IVERKSATT
+  )
+  const harIngenUavbrutteManuelleOpphoer = behandlingliste.every(
+    (behandling) =>
+      behandling.behandlingType !== IBehandlingsType.MANUELT_OPPHOER || behandling.status === IBehandlingStatus.AVBRUTT
+  )
```

Comment:
Disse to kan kanskje flyttes til `felles/utils` - der er det en del like type funksjoner som sjekker på behandlingsstatus